documentation/CONFIG.md

```diff
@@ -518,6 +519,19 @@ You can tweak some settings of the tracker to optimize OpenDataCam better for yo
 
 NB: if the object has changed ID in the past frames, it will take the last past frame known with the same ID.
 
+#### Monitor settings
+
+```json
+"MONITOR_SETTINGS": {
+    "retry_delay_ms":30,
+    "max_retries_seconds":180
+}
+```
+
+- `retry_delay_ms`: If a monitored service fails, i.e. Darknet YOLO, how long, in milliseconds, should ODC wait before trying to connect again
```

Comment:
This feels incorrect as e.g. the Mongo Service reconnection is handled differently.